### PR TITLE
ports paper crafting book and paper related items from hippie (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr https://github.com/HippieStation/HippieStation/pull/12741
"add: New origami crafting book that allows you to craft items exclusive to the book (sprited by Mr. Epic Sax)
add: new item: paper house
add: new item: paper sword
add: new item: paper throwing star
add: new item: paper handcuffs (shittiest handcuffs in the game now)
add: new item: paper identification card (starts with maint access)"
#### Changelog

:cl:  
rscadd: Adds various new paper items and a crafting book from them obtainable in uplink
imageadd: ports sprites 
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
fuck you compile tests any help is appreciated

**Comment by Hopekz** [comment]
So why is this an uplink item?
Even in the place where you ported it from its a spawn for the curator since the items themselves are not "traitory" enough to be traitor items.

You have a paper sword that deals 4 damage and a paper throwing star which deals 4 damage.

**Comment by Marmio64** [comment]
Yeah make this just something the curator can spawn with and make it one use. Also, remove the maint access from the paper ID card.

**Comment by boodaliboo** [comment]
i made it a uplink item with the other origami book because i wasnt sure if people would like it or not and if people found the items balanced if another person wants me to change it i will go and do so

**Comment by MegaEmpirical** [comment]
The maintenance access thing needs to go.

**Comment by boodaliboo** [comment]
yeah that was one of the main reasons i kept it for a antag

**Comment by overlord12345** [comment]
paper airplanes made by this kit always hit the eyes and i think they deal more damage thats why its a tot item

**Comment by boodaliboo** [comment]
> paper airplanes made by this kit always hit the eyes and i think they deal more damage thats why its a tot item

this is given alongside it now

File: code/modules/crafts/origami.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	desc = "Handcuffs made out of paper. Do these even work?"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "papercuff"
+	slot_flags = ITEM_SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 2
+	throw_range = 3
+	materials = list(/obj/item/paper)
+	breakouttime = 50 //Deciseconds = 60s = 1 minute
```
Comment (alexkar598) [inline code/modules/crafts/origami.dm]:
```suggestion
	breakouttime = 5 SECONDS
```

**Review by alexkar598 — APPROVED** [review]

**Comment by MegaEmpirical** [comment]
Does this still have the maintenance access shit?

**Comment by boodaliboo** [comment]
i have lost my motivation and i need to take a break this is negatively impacting me

**Pull request closed without merging**